packages/features/users/components/UserTable/ChangeUserRoleModal.tsx

```diff
@@ -0,0 +1,27 @@
+import { useSession } from "next-auth/react";
+import type { Dispatch } from "react";
+
+import MemberChangeRoleModal from "@calcom/features/ee/teams/components/MemberChangeRoleModal";
+
+import type { Action, State } from "./UserListTable";
+
+export function ChangeUserRoleModal(props: { state: State; dispatch: Dispatch<Action> }) {
+  const { data: session } = useSession();
+  const orgId = session?.user.organizationId;
+  if (!orgId || !props.state.changeMemberRole.user) return null;
```

Comment:
For now we only use orgs for this table - in future PR will switch this out for context to pull team Id through.